packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/PredefinedMetadataModal.tsx

```diff
@@ -0,0 +1,196 @@
+import { TypenameEnum } from "@balancer-pool-metadata/schema";
+import * as Accordion from "@radix-ui/react-accordion";
+import * as Dialog from "@radix-ui/react-dialog";
+import { useState } from "react";
+
+import { Button } from "#/components";
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "#/components/Accordion";
+import { Checkbox } from "#/components/Checkbox";
+import { usePoolMetadata } from "#/contexts/PoolMetadataContext";
+import { toSlug } from "#/utils/formatStringCase";
+
+interface Itemplate {
+  name: string;
+  options: {
+    name: string;
+    type: (typeof TypenameEnum.enum)[keyof typeof TypenameEnum.enum];
+  }[];
+}
```

Comment:
No need to define this interface if you add `as const` after the `ATTRIBUTES_TEMPLATE` definition